.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Migrate Python code to Black
```

Comment:
What does this file do?

Reply:
I left a note in the commit that formats the code on usage, you can pass it to git-blame so it ignores that particular commit. This way, if you're blame'ing around and it points at the reformat commit, you can look at that, get instructions on how to ignore it, and continue looking at blame info as if the commit never existed. Git documentation is https://git-scm.com/docs/git-blame#Documentation/git-blame.txt---ignore-revs-fileltfilegt

Reply:
OK, got it.  Nice.  Never used that functionality on git blame.